controller/src/main/java/org/jboss/as/controller/PersistentResourceXMLDescription.java

```diff
@@ -174,6 +174,7 @@ private void parseGroup(XMLExtendedStreamReader reader, ModelNode op, boolean wi
             if (reader.nextTag() == START_ELEMENT && attributeElements.containsKey(reader.getLocalName())) {
                 AttributeDefinition ad = attributeElements.get(reader.getLocalName());
                 ad.getParser().parseElement(ad, reader, op);
+                ParseUtils.requireNoContent(reader);
```

Comment:
Why this? Does it fix a problem detected by your new tests? 

What happens if the attribute group includes > 1 element attribute? AIUI parseAttributes will only parse xml attributes (because attributeGroups.isEmpty() is false; see L201 below), so parsing of element attributes is either done here or when this method returns and the loop in parseAttributeGroups continues. So if this fails if there is > 1 element attribute in a group, there doesn't seem to be way to have that kind of structure.